acm/odh-edge/apps/bike-rental-app/bike-rental-inference-generator-cronjob.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: bike-rental-inference-generator
+spec:
+  schedule: "*/1 * * * *"  # This will run the job every hour
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: test-mlflow-container
+            image: quay.io/devguyio/curl-jq
```

Comment:
Do you have a source repo for this image? It might be useful to have in a comment

Reply:
I added the Dockerfile of that, which is now called `ab-jq`